extensions/data-plane/data-plane-api/src/main/java/org/eclipse/edc/connector/dataplane/api/DataPlaneApiExtension.java

```diff
@@ -38,17 +48,37 @@
 @Extension(value = DataPlaneApiExtension.NAME)
 public class DataPlaneApiExtension implements ServiceExtension {
     public static final String NAME = "Data Plane API";
+    public static final int DEFAULT_PUBLIC_PORT = 8185;
+    public static final String PUBLIC_API_CONFIG = "web.http.public";
     @Setting
     private static final String CONTROL_PLANE_VALIDATION_ENDPOINT = "edc.dataplane.token.validation.endpoint";
```

Comment:
Maybe in another PR, but could we also use the `ControlPlaneApiUrl` for the token validation?